java/server/src/org/openqa/selenium/grid/server/BaseServerFlags.java

```diff
@@ -53,6 +53,18 @@
   @ConfigValue(section = "server", name = "allow-cors")
   private boolean allowCORS = false;
 
+  @Parameter(description = "Whether the Selenium server should communicate over https", names = "--https")
```

Comment:
It would seem that having either `--http-private-key` or `--http-certificate` implies that the connections should be secured....